tox.ini

```diff
@@ -68,10 +68,11 @@ commands =
 [testenv:integration]
 description = Run integration tests
 deps =
-    pytest
     juju
-    pytest-operator
     mysql-connector-python
+    pytest
+    pytest-operator
+    pytest-order
```

Comment:
you might need to add [this from marcelo PR](https://github.com/canonical/postgresql-k8s-operator/pull/20/files) here also